modules/olm-installing-from-operatorhub-using-cli.adoc

```diff
@@ -71,39 +71,42 @@ Note the catalog for your desired Operator.
 $ oc describe packagemanifests <operator_name> -n openshift-marketplace
 ----
 
-. An Operator group, defined by an `OperatorGroup` object, selects target namespaces in which to generate required RBAC access for all Operators in the same namespace as the Operator group.
+. If the Operator you intend to install uses the `AllNamespaces` install mode, skip this step, because the `openshift-operators` namespace already has an appropriate Operator group in place by default, called `global-operators`.
 +
-The namespace to which you subscribe the Operator must have an Operator group that matches the install mode of the Operator, either the `AllNamespaces` or `SingleNamespace` mode. If the Operator you intend to install uses the `AllNamespaces` mode, the `openshift-operators` namespace already has the appropriate `global-operators` Operator group in place.
+If the Operator you intend to install uses the `SingleNamespace` install  mode and you do not already have an appropriate Operator group in place, you must create one by following these steps:
```

Comment:
What if the Operator supports both `AllNamespaces` and `SingleNamespace` install modes?

```
      Display Name:  Couchbase Operator
      Install Modes:
        Supported:  true
        Type:       OwnNamespace
        Supported:  true
        Type:       SingleNamespace
        Supported:  false
        Type:       MultiNamespace
        Supported:  true
        Type:       AllNamespaces
```

Reply:
It's more about which of the supported methods you are choosing to utilize during this particular installation; I will sharpen this up, thx!